tendrl/ceph_integration/ceph.py

```diff
@@ -380,9 +380,14 @@ def rbd_command(command_args, pool_name=None):
 
     p = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
                          stdin=open(os.devnull, "r"))
+    if p.poll() is None: # Force kill if process is still alive
```

Comment:
Our sync runs every 3 seconds and rbd commands are pretty faster so kept the wait as 1 second here, and by that time if its not responding I terminate.
To be safer if you want we can make this at least 2 seconds. What you feel?